### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by JayKan**
fixing typo

### CODE REVIEW

File: src/Observable.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ export class Observable<T> implements Subscribable<T> {
         } else {
           // if there is NO subscription, then we're getting a nexted
           // value synchronously during subscription. We can just call it.
-          // If it errors, Observable's `subscribe` imple will ensure the
+          // If it errors, Observable's `subscribe` imply will ensure the
```
Comment (kwonoj) [inline src/Observable.ts]: actually, do we need `imply`? just `subscribe` will ensure.. maybe sufficient?
/cc @staltz

Comment (staltz) [inline src/Observable.ts]: I didn't write this text, but yes, in my opinion it can be removed.

Comment (kwonoj) [inline src/Observable.ts]: oh, sorry, thought you wrote those @staltz . @JayKan - would you mind to update PR to remove those words?

Comment (JayKan) [inline src/Observable.ts]: @kwonoj, I have submitted a new PR(#1843) that addressed this fix

Comment (jayphelps) [inline src/Observable.ts]: @JayKan sorry to make you do it, but can you instead modify this PR's commit (`git commit --amend`) so we don't have unnecessary commits and history?

**Comment by jayphelps** [comment]
Closed in favor of #1843

**Pull request closed without merging**